v2/internal/testutils/testutils.go

```diff
@@ -360,6 +360,7 @@ func (c CincinnatiMock) CincinnatiHandler(w http.ResponseWriter, r *http.Request
 	}
 }
 
+// nolint: unused
```

Comment:
This func is not being called in the code. It is safe to remove it.

Reply:
Has nothing to do with Collectors, this is mainly a function that converts from RelatedImages to CopySchema. I have changed the names to **ConvertXXX**